.travis.yml

```diff
@@ -5,3 +5,27 @@ cache:
 node_js:
   - "0.11"
   - "0.10"
+
+# Create docker image on merge to develop
+sudo: required
+dist: trusty
+services: [docker]
+env:
+  global:
+      - IMAGE_NAME=praekeltfoundation/go-jsbox-malaria24:develop
+      - REGISTRY_USER=praekeltorgdeploy
+      # REGISTRY_PASS
+      - secure: "bL5h7qWeh6zw/Gkp7IZBdcHVV0wjUPMWS/L2F4gZ0oN/V4sCKTP1vvVui8zICYoR8Ok2A5t58389s/MgQgG/lfQA0qGlLlJ6yKgypq6N2uqwbbW/Oq9sVrWvz0AVjKOqxClft/wn3shJ5lUUxeJHNdPUaCjDr+gc624V0bnKpQQhG8sAQjX30pjiW61368HbPw8pAgyGw+3nuMBffP0Eib6rtFmmUb1WQ8qdXgjiaGT9whUpohqEFY2gmRfIJzW+YTgKP+w+nlaq05zoqwujsn/FptLm/TCvlNK0iv5TOgPNOXC4QDtWaEEzb1q0uq3Rf4dywAhqJorIT7wK/coFblnQ5LSnwBSovzGdQTF52R2IDuKAjccUYZnptbGvRLLTrTGFdmA1xnS1BXc+AXJE6JTOQCpwEgixTpXYvumOIF6NBPJ+gTgomV4l743mexrKYbfmbdOc7i/ZtALQQbj6tFT2O1q/mciddzwnJA/2ddDBg9vp6+Algu0wvIcShhV1Hzf43Gx6wQCmPK5pmG4wdAEjSkNowidSC44f/dZnUajbwK5yPrju62Z2OgFh1Vvx8xzeDVO6ZobfSsH8b2ciiyz73j5QWpZZDqnZa40a890ziMiox4m4kkkcP1FXTF7v4xLbq6oWwDQJJOCoQc4wxX7Y/r26/gbr/yv3pTjtBnw="
+before_script:
+  - docker pull "$IMAGE_NAME" || true
+script:
+  - docker build --tag "$IMAGE_NAME" --cache-from "$IMAGE_NAME" .
```

Comment:
This means that we'll be overriding the default script, so we won't actually be running any of the tests anymore.

Maybe we should have 3 builds for this repo, 1 for node 0.11, one for node 0.10, and one for building the docker image.